docs/markdown/glossary.md

```diff
@@ -45,27 +44,27 @@ See instead: [Router](#router)
 
 ## Cyclic Transaction  
 
-A transaction where the destination account is the same account, on the same ledger, as the source account. This can be useful when rebalancing liquidity (to enable future payments), or when rebalancing stored value (to spread risk, or to take advantage of changing exchange rates).                                                                      
+A transaction where the destination account is the same account on the ledger as the source account. This can be useful when rebalancing liquidity (to enable future payments), or when rebalancing stored value (to spread risk, or to take advantage of changing exchange rates).                                                                      
 
 ## Destination Account 
 
 The account of the receiver whose address is included in the Interledger packet. Note that anyone can claim to have a certain Interledger address; address ownership is not enforced.                                                                       
 
 ## Destination Amount  
 
-The amount to be received by the receiver.                                                                      
+The amount to be received by the receiver through ILP.                                                                      
```

Comment:
Since ILP is itself a term, and we haven't introduce that term, we should link to it from here.

Reply:
Fixed.